[2025-10-29T00:01:33.656+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.PublishKafka scheduled__2025-10-29T00:00:00+00:00 [queued]>
[2025-10-29T00:01:33.666+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.PublishKafka scheduled__2025-10-29T00:00:00+00:00 [queued]>
[2025-10-29T00:01:33.666+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-29T00:01:33.679+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): PublishKafka> on 2025-10-29 00:00:00+00:00
[2025-10-29T00:01:33.688+0000] {standard_task_runner.py:60} INFO - Started process 574 to run task
[2025-10-29T00:01:33.692+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'PublishKafka', 'scheduled__2025-10-29T00:00:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmp2reh1xk2']
[2025-10-29T00:01:33.695+0000] {standard_task_runner.py:88} INFO - Job 128: Subtask PublishKafka
[2025-10-29T00:01:33.757+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.PublishKafka scheduled__2025-10-29T00:00:00+00:00 [running]> on host 9680281a20bc
[2025-10-29T00:01:33.867+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='PublishKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-29T00:00:00+00:00'
[2025-10-29T00:01:33.870+0000] {dag1_kafka_processing.py:122} INFO - Aucune donnée à publier.
[2025-10-29T00:01:33.870+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-29T00:01:33.883+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=PublishKafka, execution_date=20251029T000000, start_date=20251029T000133, end_date=20251029T000133
[2025-10-29T00:01:33.945+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-29T00:01:33.965+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
